streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/ImplicitConversions.scala

```diff
@@ -60,24 +60,47 @@ object ImplicitConversions {
 
   implicit def tuple2ToKeyValue[K, V](tuple: (K, V)): KeyValue[K, V] = new KeyValue(tuple._1, tuple._2)
 
-  // we would also like to allow users implicit serdes
-  // and these implicits will convert them to `Grouped`, `Produced` or `Consumed`
+  // we would also like to allow users implicit serdes and optional repartition topic / state stores names
+  // and these implicits will convert them to `Grouped`, `Produced`, `Consumed`, `Materialized` and `Joined`
+
+  implicit def groupedFromNameAndSerde[K, V](repartitionTopicName: String)
+                                            (implicit keySerde: Serde[K], valueSerde: Serde[V]): Grouped[K, V] = {
+    Grouped.`with`[K, V](repartitionTopicName)
+  }
 
-  implicit def groupedFromSerde[K, V](implicit keySerde: Serde[K], valueSerde: Serde[V]): Grouped[K, V] =
+  implicit def groupedFromSerde[K, V](implicit keySerde: Serde[K], valueSerde: Serde[V]): Grouped[K, V] = {
     Grouped.`with`[K, V]
+  }
```

Comment:
These are redundant braces we don't need to add them.

`gradle spotlessApply` will solve it all for you.